Zend/zend.h

```diff
@@ -351,6 +351,22 @@ ZEND_API void zend_save_error_handling(zend_error_handling *current);
 ZEND_API void zend_replace_error_handling(zend_error_handling_t error_handling, zend_class_entry *exception_class, zend_error_handling *current);
 ZEND_API void zend_restore_error_handling(zend_error_handling *saved);
 
+typedef void (*zend_error_notify_cb)(int type, const char *error_filename, const uint32_t error_lineno, zend_string *message);
```

Comment:
Here and elsewhere, do not use `const uint32_t`, just `uint32_t`. This makes particularly little sense inside signatures.

Reply:
zend_error_cb also has `const uint32_t` in signature, thats where I copied it from. Probably good to have another PR to clean that up, because it affects at least ext/soap